proto/umee/historacle/v1/tx.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+package umee.historacle.v1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/msg/v1/msg.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/historacle/types";
+option (gogoproto.goproto_getters_all) = false;
+
+// Msg defines the historacle Msg service.
+service Msg {
+    // StampPeriod defines the interface for which governance
+    // can use to set the stamp period.
+    rpc UpdateStampPeriod(MsgUpdateStampPeriod)
+        returns (MsgUpdateStampPeriodResponse);
+  }
+
+
+// MsgUpdateStampPeriod represents a message for governance to update the
+// stamp period.
+message MsgUpdateStampPeriod {
```

Comment:
Is is possible we would want to add a second stamp period (ex: 7 day median and 30 day median). nit: It slightly bothers me  that this is called `MsgUpdateStampPeriod` and updates the stamp_period and prune_period.

Reply:
I think for now we can keep it as a single stamp period, but yeah this should be renamed

Reply:
15b70efb8230123a7e58873b98da247fec2872ea